src/index.html

```diff
@@ -12,7 +13,40 @@
       href="styles/index.scss"
```

Comment:
The `href` attribute of the `link` tag should point to a CSS file, not an SCSS file. You need to compile your SCSS files into CSS and then link the CSS file in your HTML.